core/schema/modulesource.go

```diff
@@ -924,7 +924,7 @@ func (s *moduleSchema) collectCallerLocalDeps(
 				return nil, fmt.Errorf("error unmarshaling config at %s: %s", configPath, err)
 			}
 
-		case strings.Contains(err.Error(), "no such file or directory"):
+		case strings.Contains(err.Error(), "no such file or directory") || strings.Contains(err.Error(), "not found"):
```

Comment:
Yeah this is fun :smile: there's just different error message strings depending on whether the client is linux/windows.

I can't think of a neat way round this really.